XVim/DVTSourceTextView+XVim.m

```diff
@@ -165,9 +165,9 @@ - (void)xvim_drawRect:(NSRect)dirtyRect{
         }else{
             [self xvim_clearHighlightText];
         }
-        
-        [self xvim_drawRect:dirtyRect];
-        
+
+        [self xvim_drawRect:[self visibleRect]];
```

Comment:
Using dirtyRect here causes our gutter highlighting to be completely broken. VisibleRect fixes this with no noticeable performance decrease. In my tests.